bayesian/utils.py

```diff
@@ -801,6 +802,25 @@ def select_latest_version(latest_version='', libio_version=''):
     return return_version
 
 
+def fetch_pom_from_github(url):
+    """Fetch pom.xml from github url."""
+    base_url = 'https://raw.githubusercontent.com'
+    branch = 'master'
+    filename = 'pom.xml'
+    try:
+        if not url.startswith('https://'):
```

Comment:
Its better to consider different combinations such as [git://, git+https://, https://] or even better to get the necessary details (repoName, UserName, etc) from the url backwards.

Reply:
Take a look at [worker's parse_gh_repo()](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/utils.py#L522)